File: 3 КУРС/2 семестр/Java/main/java/com/example/voiceassistent/NumberService.java
package com.example.voiceassistent;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class NumberService {
    public static NumberApi getApi() {
        Retrofit retrofit = new Retrofit.Builder()
                .baseUrl("https://htmlweb.ru") //Базовая часть адреса
                .addConverterFactory(GsonConverterFactory.create().create()) //Конвертер, необходимый для преобразования JSON'а в объекты
                .build();

        return retrofit.create(NumberApi.class); //Создание объекта, при помощи которого будут выполняться запросы
    }

}

File: 3 КУРС/2 семестр/Учебная практика/kotlin/kotlin/app/src/main/java/com/example/kotlinapp/NumberConversionAPI/NumberService.kt
package com.example.kotlinapp.NumberConversionAPI

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory


object NumberService {
    val apiNum: NumbersAPI
        get() {
            val retrofit = Retrofit.Builder()
                .baseUrl("https://htmlweb.ru")
                .addConverterFactory(GsonConverterFactory.create())
                .build()
            return retrofit.create(NumbersAPI::class.java)
        }
}